src/win/util.c

```diff
@@ -1380,6 +1380,12 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = uv_malloc(1);
```

Comment:
Sorry for not commenting earlier... why are we doing this instead of setting it to NULL just like shell? For cross platform consistency IMHO it's better to have it as NULL rather than an empty string.